Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -335,8 +335,13 @@ open class RouteController: NSObject {
         }
     }
     
-    /// updateRouteLeg is used to notify nav-native of the developer changing the active route-leg.
-    private func updateRouteLeg(to value: Int, completionHandler: AdvanceLegCompletionHandler? = nil) {
+    /**
+    Updates current `RouteProgress.legIndex` to specified leg index.
+    
+    - parameter completionHandler: Completion handler, which is called to report a status whether
+    `RouteLeg` was changed or not.
+    */
+    public func updateRouteLeg(to value: Int, completionHandler: AdvanceLegCompletionHandler? = nil) {
```

Comment:
In theory, now we can also add one more unit test with explicit call of `updateRouteLeg(to:completionHandler:)` and `navigatorSpy.passedLegIndex` check